sdstore/sdstore.go

```diff
@@ -81,9 +99,30 @@ func handleResponse(res *http.Response) ([]byte, error) {
 	return body, nil
 }
 
-// putFile writes a file at filePath to a url with a PUT request. It streams the data
-// from disk to save memory
-func (s *sdUploader) putFile(url *url.URL, bodyType string, filePath string) error {
+// GET request from SD Store
+func (s *sdStore) get(url *url.URL) ([]byte, error) {
```

Comment:
hmm, isn't this on the store-cli? I think for `getFile` we can just use something like `io.Copy` from the response body of the GET request

Reply:
We have `put` in this file so also add `get` here.